MetaMask/metamask-extension
# Update blockaid dependency and ppom.js

## **Description**
The PR Updated `@blockaid/ppom-release` dependency and also updates `ppom.js`

## **Manual testing steps**

_1. Run extension
_2. Go to test dapp and send a malicious confirmation request
_3. Ensure that confirmation page shows blockaid error

## **Related issues**

_Fixes #???_

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
